src/encoded/search.py

```diff
@@ -786,7 +795,7 @@ def search(context, request, search_type=None, return_generator=False):
 
     # Stream response using chunked encoding.
     # XXX BeforeRender event listeners not called.
-    app_iter = iter_long_json('@graph', graph, **result)
+    app_iter = iter_long_json('@graph', graph, result)
```

Comment:
**result treats the contents of result as if it were individual parameters, but result treats it as a single object.  The effects on this function is that the individual parameters can be returned with a yield, thus passing the memory out of the function and potentially relieving the memory burden earlier.  However, result is not large AND I need to change one parameter of result based upon the contents of iterator (@graph exp list).  The memory efficiency is not needed but the memory access late in the function IS needed.